### DISCUSSION TITLE
Just moved from OZW to z-wave-js how can i give information from devices I have that is not working

### DISCUSSION THREAD

**Discussion by pjator** [zwave-js/zwave-js#2668] [Q&A]
Hello,

Sorry to say my coding skills are not what it should be but I can copy/paste and understand what i am doing =)

But after moving from OZW to z-wave-js I have the following devices that do not work:
1. Fibaro Door/windows Sensor 2 - FGDW-002
2. Fibaro Motion Sensor - FGMS-001
3. Aeotec MultiSensor 6 - ZW100-C

All other stuff worked great. 

So i guess it is a stupid question but how can I help to make these devices available, is the a standard template I can use to fill in data from the device manuals (or copy some similar product that works). Or is there a way I can extract data from these devices that can be used for someone with more skills but don't have the device. 

Tried to google but no solutions from what I can see so try to add my first comment here on Github. 

/P

**Comment by pjator** [comment]
I understand that I should most likely have to do a PR but want to make sure I am constructive when I do so since I have not done a PR before.

**Reply by joselito11** [reply]
Fibaro motion sensors are working great, wake up and re-interview should do it.

**Reply by pjator** [reply]
Then I must be doing something wrong because I have removed and added the device to the network without success it gives no information about the device you are talking about the FGMS-001

**Reply by pjator** [reply]
Okey, after a restart of z-wave-js the node just started to work.. strange.
then two left =)

    Fibaro Door/windows Sensor 2 - FGDW-002
    Aeotec MultiSensor 6 - ZW100-C

**Reply by pjator** [reply]
A well now the Aeotec Multisensor started to work as well... sorry I pressume the door sensor will soon wake up as well. So please discard this tread. Sorry for disturbing looks like you all have done a great work with this!

**Reply by AlCalzone** [reply]
Battery powered devices need to be woken up manually, so zwave-js can figure out their capabilities. It sounds like the initial interview didn't complete, so you were waiting for them to wake up on their own.

> So i guess it is a stupid question but how can I help to make these devices available

We have an entire section on this in the documentation: https://zwave-js.github.io/node-zwave-js/#/config-files/overview and the following links under "Device Configuration Files" in the sidebar.

**Reply by pjator** [reply]
(I am not this stupid =)

I was fully aware I needed to wake the devices up and hade both woken them up and also tried to remove and rejoin the network without success and did not find them in the z-wave-js database. 

But all of a sudden the devices started to work and have now worked perfect during sunday. 
So just problem with the initial interview. So can just  say thanks again for a fantastic update to Home Assistant the speed of this compared to OZW is clearly noticable. 
/P

**Reply by AlCalzone** [reply]
> I was fully aware I needed to wake the devices up

Trust me, I can never be too careful with this. I lost count how often I've heard that users have "woken" the device only to find out later that they triggered the motion sensor or something instead.
